README.md

```diff
@@ -26,6 +39,109 @@
 - [Local Development](#local-development)
 - [LICENSE](#license)
 
+
+
+<!-- - [Supported Browsers](#supported-browsers)
+- [Packages](#packages)
+  - [Versioning](#versioning)
+  - [Status](#status)
+  - [Deprecated](#deprecated)
+- [Supported Browsers](#supported-browsers)
+- [Internationalization (I18n)](#internationalization-i18n)
+  - [Packages Requiring I18n](#packages-requiring-i18n)
+- [Contributing](#contributing)
+- [Local Development](#local-development)
+- [LICENSE](#license) -->
+
+<hr>
+<!--  -->
+
+## Supported Browsers
+
+| Browser                     | Version |
+|-----------------------------|---------|
+| Chrome & Chrome for Android | Current |
+| Edge                        | Current |
+| Firefox                     | Current |
+| Internet Explorer           | 10 & 11 |
+| Safari & Mobile Safari      | Current |
+
+<hr>
+<!--  -->
+
+## Getting Started
+
+### Dependencies
+-   D3
+-   Polyfills:
+    -   Object.assign
+    -   Object.values
+
+### Install
+
+You can install Carbon via `npm` or download the [latest release](https://github.com/cerner/terra-graphs/releases/latest) from GitHub.
+
+```sh
+npm i @cerner/carbon-graphs --save-dev
+```
+
+<!-- ```sh
+npm i @cerner/Terra Graphs-graphs --save-dev
+``` -->
+
+### Getting Started
+
+-   [Getting Started](./packages/terra-graphs/docs/getting-started/GettingStarted.md)
+-   [API Reference](docs/README.md)
+
+<hr>
+<!--  -->
+
+## Features
+
+-   Adheres to Cerner standard design template
+-   Responsive
+-   Native-built graphs
+-   Default theme for data points
+
+### Graph types
+
+-   [Line](./packages/terra-graphs/docs/controls/Line.md#usage)
+-   [Multi Line](./packages/terra-graphs/docs/controls/Line.md#multi-line)
+-   [Spline Line](./packages/terra-graphs/docs/controls/Line.md#spline-line)
+-   [Paired Result](./packages/terra-graphs/docs/controls/PairedResult.md#usage)
+-   [Bar](./packages/terra-graphs/docs/controls/Bar.md#usage)
+-   [Timeline](./packages/terra-graphs/docs/controls/Timeline.md#usage)
+-   [Pie](./packages/terra-graphs/docs/controls/Pie.md#usage)
+-   [Gantt](./packages/terra-graphs/docs/controls/Gantt.md#usage)
+-   [Scatter](./packages/terra-graphs/docs/controls/Scatter.md#usage)
+-   [Bubble](./packages/terra-graphs/docs/controls/Bubble.md#usage)
+    -   [Bubble Single Dataset](./packages/terra-graphs/docs/controls/BubbleSingleDataset.md)
+    -   [Bubble Multiple Dataset](./packages/terra-graphs/docs/controls/BubbleMultipleDataset.md)
```

Comment:
These documentation links are not working in the Terra -Deployment link
https://terra-graphs-migration-co6hr5o.herokuapp.com/home

![image](https://user-images.githubusercontent.com/62761486/97498125-9f37da00-1939-11eb-9c92-f795fb07a6ed.png)

Reply:
Hmm, the links are working locally and are similar to the ones in the original carbon-graphs repo.